mqtt-transport-binding.md

```diff
@@ -215,10 +214,10 @@ source: /mycontext/subcontext
 
 ### 3.2. Structured Content Mode
 
-The _structured_ content mode keeps event metadata and data together in the
-payload, allowing simple forwarding of the same event across multiple routing
-hops, and across multiple transports. This is the only supported mode for MQTT
-3.1.1
+The _structured_ content mode keeps event metadata and data payload together in
+the payload, allowing simple forwarding of the same event across multiple
```

Comment:
See `event data` discussion in `spec.md`.

This sentence is confusing now: `keeps [...] data payload in the payload`?

I recommend against this change.